### DISCUSSION TITLE
[Imp]Streaming with Promptflow endpoint

### DISCUSSION THREAD

**Discussion by aashish181188** [microsoft/sample-app-aoai-chatGPT#984] [Q&A]
Hi All,
We are using promptflow endpoint in chatbot web app . Currently there is no code to support streaming from promptflow  in web app code in this repository.

I have written logic to handle the streaming from promptflow but I am facing challenge to display streaming on user interface written in reactjs as the UI code only supports non stream response.
Can anyone guide on how to display streaming on user interface using reactjs in existing git repository. 

Thanks!!

**Comment by OlegDStassenko** [comment]
Hello, I’m facing the same issue. Users are not satisfied because they feel the wait time is too long. How can we display the stream in the application when using Prompflow?
